src/ero_mmnet.py

```diff
@@ -106,6 +113,39 @@ def import_circles(self, ego_node_id, ego_network_circles_path):
 
             self.circles[ego_node_id] = circles
 
+    def import_events(self, events_filename):
+        parser = event_parser.EventParser()
+        events_json = json.load(open(events_filename))['data']
+        events_mode = self.mmnet.GetModeNetByName("Event")
+        for event in events_json:
+            parsed_event = parser.parse_event(event)
+            self.events[parsed_event.id] = event
+            events_mode.AddNode(parsed_event.id)
+            self.connect_event_in_network(parsed_event)
+
+    def connect_event_in_network(self, event):
+        neighbors = self.generate_random_neighbors()
+        crossnet_person_to_event = self.mmnet.GetCrossNetByName(
+            "EventToPerson")
+        for neighbor in neighbors:
+            edge_id = crossnet_person_to_event.AddEdge(event.id, neighbor)
+            self.event_to_person_edges[edge_id] = (
+                event.id, neighbor)
+
+    def generate_random_neighbors(self):
+        persons = self.people.values()
+        person_max_index = len(persons) - 1
+        edge_count = int(round(numpy.random.normal(self.EVENT_PERSON_EDGES_MU, self.EVENT_PERSON_EDGE_SIGMA, 1)))
+        neighbor_ids = []
+        while len(neighbor_ids) < edge_count:
+            random_neighbor_index = numpy.random.randint(0, person_max_index)
```

Comment:
I don't see any reason why max_index should be len(persons)-1 when randint(a,b) returns a number in [a,b[, hence where b is excluded.